common/djangoapps/microsite_configuration/backends/base.py

```diff
@@ -0,0 +1,160 @@
+"""
+Microsite configuration backend module.
+
+Contains the base class for microsite backends.
+
+"""
+
+from __future__ import absolute_import
+
+import abc
+import edxmako
+import os.path
+
+from django.conf import settings
+
+
+# pylint: disable=unused-argument
+class BaseMicrositeBackend(object):
```

Comment:
We can also, move the implementation of this method to the `SettingsFileMicrositeBackend` and thus make this class really abstract